GGXrdReversalTool/Controls/EventControl.xaml.cs

```diff
@@ -14,7 +14,12 @@ public EventControl()
         InitializeComponent();
     }
 
-    public IEnumerable<object> ActionTypes => new string[]{"Animation", "Combo", "Simulated roundstart"};
+    public IEnumerable<object> ActionTypes => new string[]{
```

Comment:
It seems like the purpose of this property is databinding with the combo box. But I think that IEnumerable of an Enum type is a better.
If you get troubles to do that. Just add a TODO and i'll do it later